emscripten/Markerless/include/PatternDetector.hpp

```diff
@@ -29,8 +29,9 @@ class PatternDetector
     PatternDetector
         (
         cv::Ptr<cv::FeatureDetector>     detector = cv::ORB::create(1000),
-        cv::Ptr<cv::DescriptorExtractor> extractor = cv::xfeatures2d::FREAK::create(false, false),
-        cv::Ptr<cv::DescriptorMatcher>   matcher   = cv::BFMatcher::create(cv::NORM_HAMMING, true),
+        //cv::Ptr<cv::DescriptorExtractor> extractor = cv::xfeatures2d::FREAK::create(false, false),
```

Comment:
i change this as i made in my MasteringOpenCV fork this [commit](https://github.com/kalwalt/MasteringOpenCV/commit/ed11e1bf6badc919771606cfd2f7b3ad8fc232df)